extension/chrome/elements/pgp_block.ts

```diff
@@ -106,7 +113,117 @@ export class PgpBlockView extends View {
   };
 
   public setHandlers = () => {
-    // defined as needed, depending on what rendered
+    $('.pgp_print_button').click(this.setHandler(() => this.printPGPBlock()));
+  };
+
+  private initPrintView = async () => {
```

Comment:
These two new methods could go into `PgpBlockViewRenderModule` since they mostly deal with rendering things. If the render module has `setHandlers` there as well, the handler could probably go there to stay close to the moved code.